jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -71,6 +71,9 @@
 	static final boolean enableCompression = com.ibm.oti.vm.VM.J9_STRING_COMPRESSION_ENABLED;
 	
 	/*[IF Sidecar19-SE]*/
+	/*[IF Java18.3]*/
+	private byte coder;
```

Comment:
This field should be final. That would highlight the missing initialization in 3 constructors.

Reply:
sigh... fixed.

Reply:
That's better. Thanks, Jason.